### PR TITLE
101 rh implementing skills list

### PR DESCRIPTION
**Pull request by rezahusain**
skills_list

Hey guys,

Sorry for the delay regarding the implementation of skills list. Originally tried to use Javascript, however it wasn't working no matter how hard I tried to debug it. Ended up using checkboxes for the user to click on with the label of each corresponding skill next to them. After sign up, each skill is added to the skills array for future usage. I didn't add any "required amount" to the amount of skills being selected since we are not using the skills for the matching algorithm. 
Also fixed some issues with the dev.rake and the registrations controller where the cohort member association wasn't being named properly.
If you guys have any requests I'm more than open to making changes to the code. Just shoot me a message!

Reza

### CODE REVIEW

**Review by MayaS1111 — APPROVED** [review]
Honestly not what I was hoping for, but I understand. Thanks

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def create
     super do |resource|
       if resource.persisted?
         Rails.logger.info "User created: #{resource.inspect}"
-        if resource.cohort_members.any?
+        if resource.cohort_member.any?
```
Comment (MayaS1111) [inline app/controllers/users/registrations_controller.rb:18]: why is this signal?

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -164,7 +164,7 @@ task({ sample_data: :environment }) do
 
   # Creating Matches
   mentees.each do |mentee|
-    cohort_member_mentee = mentee.cohort_members.first
+    cohort_member_mentee = mentee.cohort_member
```
Comment (MayaS1111) [inline lib/tasks/dev.rake:167]: why is this signal?

**Review by MayaS1111 — CHANGES_REQUESTED** [review]
Why is cohort_member association singular?

**Review by YoungSong99 — APPROVED** [review]
Thanks Reza
I think we can implement search the skills and if there is no skill user wants to put, adding the skill for next step.

**Review by MayaS1111 — APPROVED** [review]
All good now

**Pull request merged**